documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/bridge/namedpredicate/SkuIdentifierBinder.java

```diff
@@ -101,17 +101,17 @@ public SearchPredicate create(NamedPredicateProviderContext context) {
 				String[] patternParts = pattern.split( "\\." );
 				if ( patternParts.length > 0 ) {
 					b.must( f.wildcard()
-							.field( context.absolutePath( "departmentCode" ) ) // <5>
+							.field( "departmentCode" ) // <5>
 							.matching( patternParts[0] ) );
 				}
 				if ( patternParts.length > 1 ) {
 					b.must( f.wildcard()
-							.field( context.absolutePath( "collectionCode" ) )
+							.field( "collectionCode" )
 							.matching( patternParts[1] ) );
 				}
 				if ( patternParts.length > 2 ) {
 					b.must( f.wildcard()
-							.field( context.absolutePath( "itemCode" ) )
+							.field( "itemCode" )
```

Comment:
Hey @wklaczynski , just a heads-up: now that this PR has been merged, you no longer need to use `absolutePath()` in named predicates when using the Search DSL. For native queries, you can still use `context.predicate().toAbsolutePath(...)` (instead of `context.absolutePath(...)` previously)